draft-ietf-rtcweb-jsep.xml

```diff
@@ -2133,10 +2133,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 6.1.
           </t>
 
-          <t>As discussed above, the following attributes are included only
-          if the m= section either has a unique address or is associated
-          with the bundle-tag: "a=ice-ufrag",
-          "a=ice-pwd", "a=fingerprint", "a=dtls-id", and "a=setup".
+          <t>As discussed above, the following attributes of category IDENTICAL
+          or TRANSPORT are included only if the data m= section either has
+          a unique address or is associated with the bundle-tag (e.g., if
+          it is the only m= section):
```

Comment:
According to the current text, you insert a bundle group regardless of the number of m sections. Or did you mean something else?